trans_tdsm.py

```diff
@@ -94,6 +96,9 @@ def train_model(files_list_, device='cpu'):
     eps_ = []
     batch_ct = 0
     for epoch in range(0, config.epochs ):
+        sys.stdout.write('\r')
```

Comment:
Not sure we want the local tracking anymore seeing as we should be able to track on WandB but I'll leave it for now